.circleci/config.yml

```diff
@@ -1,84 +1,84 @@
+                  command: echo $CIRCLE_JOB > circlejob.txt
 
-      - restore_cache:
-          key: deps1-{{ .Branch }}-{{ checksum "requirements-dev.txt" }}-{{ checksum "package.json" }}-{{ checksum "package-lock.json" }}-{{ checksum ".circleci/config.yml" }}-{{ checksum "circlejob.txt" }}
+            - restore_cache:
+                  key: deps1-{{ .Branch }}-{{ checksum "requirements-dev.txt" }}-{{ checksum "package.json" }}-{{ checksum "package-lock.json" }}-{{ checksum ".circleci/config.yml" }}-{{ checksum "circlejob.txt" }}
 
-      - run:
-          name: Install dependencies
-          command: |
-              sudo pip install virtualenv --upgrade
-              . venv/bin/activate
-              pip install -r requirements-dev.txt
-              npm install --ignore-scripts
+            - run:
+                  name: Install dependencies
+                  command: |
+                      sudo pip install virtualenv --upgrade
+                      . venv/bin/activate
+                      pip install -r requirements-dev.txt
+                      npm install --ignore-scripts
 
-      - save_cache:
-          key: deps1-{{ .Branch }}-{{ checksum "requirements-dev.txt" }}-{{ checksum "package.json" }}-{{ checksum "package-lock.json" }}-{{ checksum ".circleci/config.yml" }}-{{ checksum "circlejob.txt" }}
-          paths:
-              - "venv"
-              - "node_modules"
+            - save_cache:
+                  key: deps1-{{ .Branch }}-{{ checksum "requirements-dev.txt" }}-{{ checksum "package.json" }}-{{ checksum "package-lock.json" }}-{{ checksum ".circleci/config.yml" }}-{{ checksum "circlejob.txt" }}
+                  paths:
+                      - 'venv'
+                      - 'node_modules'
 
-      - run:
-          name: prettier --list-different
-          command: npm run format:test
+            - run:
+                  name: prettier --list-different
+                  command: npm run format:test
 
-      - run:
-          name: npm run lint
-          command: npm run lint
+            - run:
+                  name: npm run lint
+                  command: npm run lint
 
-      - run:
-          name: Build
-          command: |
-              . venv/bin/activate
-              npm run build:js
-              npm run build:py
+            - run:
+                  name: Build
+                  command: |
+                      . venv/bin/activate
+                      npm run build:js
+                      npm run build:py
 
-      - run:
-          name: Run tests
-          command: |
-              . venv/bin/activate
-              python --version
-              python -m unittest test.test_integration
-              python -m unittest test.test_dash_import
+            - run:
+                  name: Run tests
+                  command: |
+                      . venv/bin/activate
+                      python --version
+                      npm run test-unit
```

Comment:
This is the only real difference apart from formatting - running the new unit tests in CircleCI.